src/examples/FileInputForm.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component, PropTypes} from 'react';
+import {reduxForm} from 'redux-form';
+export const fields = ['avatar'];
+
+class FileInputForm extends Component {
+  static propTypes = {
+    fields: PropTypes.object.isRequired,
+    handleSubmit: PropTypes.func.isRequired,
+    resetForm: PropTypes.func.isRequired
+  };
+
+  render() {
+    const {
+      fields: {avatar},
+      handleSubmit,
+      resetForm
+      } = this.props;
+    return (<form className="form-horizontal" onSubmit={handleSubmit}>
+        <div className="form-group">
+          <label className="col-xs-4 control-label">Avatar</label>
+          <div className="col-xs-8">
+            <input type="file" className="form-control" {...avatar} value={ null } />
```

Comment:
It is mentioned at the end of the markdown file. Alternative looks like:

```
render() {
  const { fields: { avatar } } = this.props;
  // pull out value so it isn't passed into the file input
  let { value, ...avatarProps } = avatar;

  return (
    ...
      <input type="file" { ...avatarProps } />
    ...
  );
}
```

Downside is an additional unused binding (`let value = avatar.value`). Perhaps this is easier to understand than setting `value={ null }` on the input itself.